canvassyncer/__main__.py

```diff
@@ -36,13 +36,20 @@ async def downloadOne(self, src, dst):
                 if res.status_code != 200:
                     return self.failures.append(f"{src} => {dst}")
                 num_bytes_downloaded = res.num_bytes_downloaded
-                async with aiofiles.open(dst, "+wb") as f:
-                    async for chunk in res.aiter_bytes():
-                        await f.write(chunk)
-                        self.tqdm.update(
-                            res.num_bytes_downloaded - num_bytes_downloaded
-                        )
-                        num_bytes_downloaded = res.num_bytes_downloaded
+                dst_temp = dst+".temp"
```

Comment:
You need to format your code with `black`.  https://github.com/psf/black

Reply:
Formated